src/Log/LogRepository.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+namespace Give\Log;
+
+use Give\Framework\Database\DB;
+
+/**
+ * Class LogRepository
+ * @package Give\Log\Repositories
+ *
+ * @since 2.9.7
+ */
+class LogRepository {
+	/**
+	 * Insert Log into database
+	 *
+	 * @param  LogModel  $model
+	 *
+	 * @return int inserted log id
+	 */
+	public function insertLog( LogModel $model ) {
+		global $wpdb;
+
+		DB::insert(
+			$wpdb->give_log,
+			[
+				'log_type'     => $model->getType(),
+				'migration_id' => $model->getMigrationId(),
+				'data'         => $model->getData( $jsonEncode = true ),
+				'category'     => $model->getCategory(),
+				'source'       => $model->getSource(),
+			],
+			null
+		);
+
+		return $wpdb->insert_id;
```

Comment:
Go ahead and add a `DB::last_insert_id()` method to the static facade so this isn't needed.

Reply:
Resolved 79e2f03d